packages/botonic-react/src/webchat/devices/scrollbar-controller.js

```diff
@@ -57,25 +56,23 @@ export class ScrollbarController {
   }
 
   toggleOnMouseWheelEvents() {
-    const webchat = getWebchatElement()
     if (this.hasScrollbar()) {
-      webchat.onmousewheel = {}
+      this.webchat.onmousewheel = {}
       return
     }
-    webchat.onmousewheel = e => e.preventDefault()
+    this.webchat.onmousewheel = e => e.preventDefault()
```

Comment:
we should test these changes in the simulator, will try to do it in an iOS simulator 👍